apps/ejabberd/src/ejabberd_listener.erl

```diff
@@ -173,13 +174,14 @@ listen_tcp(PortIPProto, Module, SockOpts, Port, IPS) ->
             ListenSocket;
         _ ->
             Res = gen_tcp:listen(Port, [binary,
+                                        {backlog, 100},
                                         {packet, 0},
                                         {active, false},
                                         {reuseaddr, true},
                                         {nodelay, true},
                                         {send_timeout, ?TCP_SEND_TIMEOUT},
                                         {keepalive, true},
-                                        {send_timeout_close, true}]),
+                                        {send_timeout_close, true} | SockOpts]),
```

Comment:
No, they have to be at the end, I looked into the `gen_tcp`, and `inet` code. There is a recursive function iterating on this opts and construing a record with opts.